### DISCUSSION TITLE
Is it possible to revert to previous version

### DISCUSSION THREAD

**Discussion by dougalbob** [RandomNinjaAtk/docker-lidarr-extended#23] [General]
Hi

I am seeing a noticable drop in the hit rate of late.  I cant quote specific version numbers but about a week ago when I was on holiday I would say I was grabbing about 80% of releases and now I'm more like 25%.   

I did a test today with Blink 182 just to see what of the 8 studio releases it would handle and I got one succesful download.  

I then changed  the default Lidarr releases by to best match the releases on Deezer and this picked up 1 further album 

Then added Deezer URLs to each of the remaining 6 missing releases in musicbrainz  and this picked up a further 4 albums.  

Is it possible to revert to earlier version (somewhere after the incomplete downloads were Identified and fixed) ?

**Comment by RandomNinjaAtk** [comment]
What is the script version? (always make sure your on the latest to have the most recent improvements

I believe there is a way you could pull an older image, but that is not ideal. The latest version works very well in my testing. Its possible that your not on the latest and the script has changed rapidly...

**Reply by dougalbob** [reply]
Hi I'm on lidarr-extended (1.0.196)  which I think is the last but one release.  I hadnt grabbed the last one as didnt wont to risk messingup up the current folder paths which are working fine for me, 

Perhaps you could suggest an artist I could try that you have already picked up and I can see if there is any mismatch in the success rates we are seeing?

**Reply by RandomNinjaAtk** [reply]
The latest is 1.0.201, so you should start with updating. A lot of search optimizations were developed during 188-200. Some had bugs... so could be possible that your experiencing issues with a bug because your not on the latest script code.

**Reply by dougalbob** [reply]
Blimey, hadnt realised there had been so many updates pushed out in that short time!  I'll update now

**Reply by dougalbob** [reply]
Tested with a few more artists now using latest version and it is doing a fine job such that I've now disabled my indexers in Lidarr entirely.

**Reply by RandomNinjaAtk** [reply]
> Tested with a few more artists now using latest version and it is doing a fine job such that I've now disabled my indexers in Lidarr entirely. 

This is the way... 😉
